variants/DEMO_F030F4/variant.cpp

```diff
@@ -54,6 +54,9 @@ const PinName digitalPin[] = {
   PA_10, //D10 - SDA (RX UART Header)
   PA_13, //D11 - SWDIO
   PA_14, //D12 - SWCLK
+  // These two are only available on boards without a crystal:
+  PF_0,
+  PF_1,
```

Comment:
Please update comments for all Analog pins:
 PA_0,  //D13/A0 ~ D0
```patch
-PA_0,  //D13/A0 ~ D0
+PA_0,  //D15/A0 ~ D0
 ```

Reply:
👍